news/2020-03-13-featured-artist-beatmap-updates-from-the-mappers-guild.md

```diff
@@ -0,0 +1,439 @@
+
+This quest was completed by **[Cris-](https://osu.ppy.sh/users/6175280)**, **[walaowey](https://osu.ppy.sh/users/1475828)**, **[Minorsonek](https://osu.ppy.sh/users/5377178)**, **[Cosmolade](https://osu.ppy.sh/users/2172778)**, **[Gero](https://osu.ppy.sh/users/1467715)**, **[Tatan](https://osu.ppy.sh/users/5646529)**, **[Crissa](https://osu.ppy.sh/users/5405836)**, **[pkhg](https://osu.ppy.sh/users/343624)**, **[Seolv](https://osu.ppy.sh/users/8067876)** and **[TheShadowOfDark](https://osu.ppy.sh/users/5795337)**
+
+- [Loki - Quiet Village](https://osu.ppy.sh/beatmapsets/1050693#osu/2195976) hosted by [Gero](https://osu.ppy.sh/users/1467715)
+- [Loki - Wizard's Tower](https://osu.ppy.sh/beatmapsets/1064248#osu/2228263) hosted by [Minorsonek](https://osu.ppy.sh/users/5377178)
+- [Loki - Troll of the Forsaken Cave](https://osu.ppy.sh/beatmapsets/1052028#osu/2198578) hosted by [Cris-](https://osu.ppy.sh/users/6175280)
+- [Loki - With Fire and Sword (2013)](https://osu.ppy.sh/beatmapsets/1088528#osu/2276018) hosted by [TheShadowOfDark](https://osu.ppy.sh/users/5795337)
+- [Loki - The Victory is Ours](https://osu.ppy.sh/beatmapsets/1066365#osu/2232865) hosted by [Crissa](https://osu.ppy.sh/users/5405836)
+- [Loki - Interlude](https://osu.ppy.sh/beatmapsets/1091243#osu/2281346) hosted by [Tatan](https://osu.ppy.sh/users/5646529)
+
+![Mystery header](/wiki/shared/news/banners/mappersguild-mystery.jpg)
+
+For the **Mystery Electronic mini-pack (osu!taiko)** quest, the mappers had to create and rank at least 3 mapsets of unpublished electronic songs by one of our unreleased featured artists.
+
+This quest was completed by **[tasuke912](https://osu.ppy.sh/users/2774767)**, **[salchow](https://osu.ppy.sh/users/9738159)**, **[HiroK](https://osu.ppy.sh/users/4050738)**, **[Nepuri](https://osu.ppy.sh/users/6637817)**, **[Cynplytholowazy](https://osu.ppy.sh/users/3901754)** and **[Nardoxyribonucleic](https://osu.ppy.sh/users/876419)**
+
+- [MYLK - Let You Go](https://osu.ppy.sh/beatmapsets/1053583#taiko/2201615) hosted by [Nepuri](https://osu.ppy.sh/users/6637817)
+- [MYLK - Prism](https://osu.ppy.sh/beatmapsets/1051590#taiko/2197618) hosted by [salchow](https://osu.ppy.sh/users/9738159)
+- [MYLK - Smile](https://osu.ppy.sh/beatmapsets/1065517#taiko/2231210) hosted by [Nardoxyribonucleic](https://osu.ppy.sh/users/876419)
+- [MYLK - Bubblegum](https://osu.ppy.sh/beatmapsets/1055143) hosted by [Cynplytholowazy](https://osu.ppy.sh/users/3901754)
+- [MYLK - SOAP](https://osu.ppy.sh/beatmapsets/1055618#taiko/2205809) hosted by [tasuke912](https://osu.ppy.sh/users/2774767)
+
+![Mystery header](/wiki/shared/news/banners/mappersguild-mystery.jpg)
+
+For the **Mystery Rock mini-pack (osu!, osu!taiko, osu!catch)** quest, the mappers had to create and rank at least 3 mapsets of unpublished rock songs by one of our unreleased featured artists.
```

Comment:
insert paragraph about how these maps were paired with the fa announcement for Blue Stahli https://osu.ppy.sh/home/news/2020-01-22-new-featured-artist-blue-stahli